hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/systemcontracts/ExchangeRateSystemContractTest.java

```diff
@@ -100,6 +103,7 @@ void convertsPositiveNumberToTinycentsAsExpected() {
 
     @Test
     void convertsZeroToTinybarsAsExpected() {
+        given(frame.getValue()).willReturn(Wei.ZERO);
         givenRate(someRate);
 
         final var zeroInput = tinycentsInput(0);
```

Comment:
Should there be a unit test that tests the new exception case?

Reply:
Added.